ios/PurchasesHybridCommon/PurchasesHybridCommonTests/StoreProductDiscount+HybridAdditionsTests.swift

```diff
@@ -23,7 +23,8 @@ class SkuProductDiscountHybridAdditionsTests: QuickSpec {
                                                         numberOfPeriods: 3,
                                                         paymentMode: SKProductDiscount.PaymentMode.payAsYouGo,
                                                         type: .introductory)
-                guard let receivedDictionary = productDiscount.rc_dictionary as? [String: NSObject] else {
+                let storeProductDiscount = StoreProductDiscount(sk1Discount: productDiscount)!
```

Comment:
Can you use `XCTUnwrap` here?

Reply:
will do! I'll do it in the follow-up PR to prevent more merge conflicts

Reply:
done! https://github.com/RevenueCat/purchases-hybrid-common/pull/133/commits/82690acc8d22288920ef4b75338b1f5cb597eaa6